### ISSUE TITLE
arm-sonoff-linux-uclibcgnueabi-gcc

### ISSUE THREAD

**Issue by shafr** [roleoroleo/sonoff-hack#48]
Where can I find `arm-sonoff-linux-uclibcgnueabi-gcc` package (UBUNTU)? 
I cannot build firmware otherwise, due to error:

```
/bin/sh: 1: arm-sonoff-linux-uclibcgnueabi-gcc: not found
or 
/sonoff-hack/src/busybox/busybox/scripts/gcc-version.sh: line 11: arm-sonoff-linux-uclibcgnueabi-gcc: command not found
```

google does not help much.

**Comment by roleoroleo** [comment]
Check the wiki:
https://github.com/roleoroleo/sonoff-hack/wiki/Toolchain

**Comment by shafr** [comment]
OK, thank you!
I thought about adding github action that would build artifact automatically after push. 
Does it makes sense ?

**Comment by roleoroleo** [comment]
Yes.
crdnl had already done something similar on the Allwinner platform:
https://github.com/roleoroleo/yi-hack-Allwinner/actions

**Comment by shafr** [comment]
What would be perfect if you can set up docker container with tooling inside that can be used for build. 
This way github action can be run inside that container & everyone who can build it locally can use that container.

**Comment by shafr** [comment]
Ok, one more:
```
./compile.www: 29: yui-compressor: not found
```

**Comment by roleoroleo** [comment]
You need to install the package yui-compressor

**Comment by shafr** [comment]
Ok, great.
What is output artifact ? Tar'ed contents of build folder ?

**Comment by roleoroleo** [comment]
The build folder with some changes.
https://github.com/roleoroleo/sonoff-hack/blob/master/scripts/pack_fw.sh

**Comment by shafr** [comment]
Implemented build in #51
